edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/cloud/RetryingCloudProxy.cs

```diff
@@ -84,6 +85,7 @@ async Task<T> ExecuteOperation<T>(Func<ICloudProxy, Task<T>> func, string operat
                         throw;
                     }
 
+                    OperationMetrics.Instance.LogRetryOperation(1, this.id, operation);
```

Comment:
Are these the only 2 places where we retry?

Reply:
there is retry in getting scopes which I didn't and subscription processor. Do you think it would make sense to add those?